.gitignore

```diff
@@ -98,6 +98,8 @@ ENV/
 
 # mkdocs documentation
 /site
+doc/source/gallery
+doc/source/gallery_vignettes
```

Comment:
I also noticed I'm getting the following directories created, which could be added to the gitignore: 

```
docs/cold-springs-fire/
docs/gallery_vignettes/
``